adding wording around fixed remote working (#364)
madetech/handbook: benefits/remote_working.md

- Make sure you choose days that will have the least impact on customers. A larger organisation's time will be less flexible and you'll need to ensure there are clearly set expectations around when showcases will be etc.
- It is also important to ensure that a regular absence doesn't impact the productivity and success of other team members (such as a pairing partner), or interrupt the flow of regular team ceremonies.

Note: 

> For fixed remote working requirements (i.e. I need to work from home every Wednesday) there are additional considerations. We would generally want to try and faciliate this but it is dependant on the types of projects we have at that time. For customer facing roles (tech and delivery) we'll have to communicate this to clients in advance to manage their expectations. In some instances a fixed remote working requirement may not suit a particular engagement which could limit the engagements we can assign to you. 

Reviewer: ```suggestion
For fixed remote working requirements (i.e. I need to work from home every Wednesday) there are additional considerations. We would generally want to try and facilitate this, but it is dependant on the types of projects we have at that time. For customer-facing roles (tech and delivery), we'll have to communicate this to clients in advance to manage their expectations. In some instances, a fixed remote working requirement may not suit a particular engagement which could limit the engagements we can assign to you. 
```

This is a quick go-over in Grammarly. I agree with comments in Slack that the voice of this is very neutral and might not make it wholly clear whether it's OKAY that you have commitments.

> For fixed remote working requirements (i.e. I need to work from home every Wednesday) there are additional considerations. We would generally want to try make faciliate this but this will depend on the types of projects we have at that time. For customer facing roles (tech and delivery) we'll have to communicate this to clients in advance to manage their expectations. In some instances a fixed remote working requirement may not suit a particular engagement which could limit the engagements we can assign to you. 

Reviewer 2: ```suggestion
For fixed remote working requirements (i.e. I need to work from home every Wednesday) there are additional considerations. We would generally want to try and faciliate this but it is dependant on the types of projects we have at that time. For customer facing roles (tech and delivery) we'll have to communicate this to clients in advance to manage their expectations. In some instances a fixed remote working requirement may not suit a particular engagement which could limit the engagements we can assign to you. 
```

Reviewer 2: (Fixes up the wording of the second sentance)
